Fix Distributed EventList usage

sraikund16 opened the pull request:
Summary: Summarized here: https://github.com/pytorch/pytorch/issues/132227

Test Plan: Use suggestion in issue, should see test passing again

Differential Revision: D60614690


cc @XilunWu @H-Huang @awgu @kwen2501 @wanchaol @fegin @fduwjj @wz337 @wconstab @d4l3k @c-p-i-o

aaronenyeshi reviewed (approved):
Awesome!

aaronenyeshi commented:
Please double check the lint issue, trailing space: 
![image](https://github.com/user-attachments/assets/85cdb952-3ab9-40ac-8ebd-3f36c6ed81c7)

sraikund16 committed:
Fix Distributed EventList usage (#132448)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/132448

Summarized here: https://github.com/pytorch/pytorch/issues/132227

Test Plan: Use suggestion in issue, should see test passing again

Reviewed By: aaronenyeshi

Differential Revision: D60614690
